python/BioSimSpace/Align/_align.py

```diff
@@ -97,6 +98,14 @@ def generateNetwork(molecules, names=None, work_dir=None, plot_network=False,
            If using a 'work_dir', then a PNG image will be located in
            'work_dir/images/network.png'.
 
+       scores_file : str
+           Path to a CSV file in the form:
+           lig1,lig2,score
+           ..
+           lig_ lig_,score
+           , where score is a float that is fed into LOMAP for edge scoring
+           (instead of the default LOMAP score based on MCSS).
```

Comment:
Can we say "Maximum Common Substructure (MCS)" here, just in case the user doesn't know what the acronym means. (Unlikely, but it's good to be explicit.)

Reply:
see f93dfa096da5bb4be09259edeab90a2f1f77c767